cbrain_task_descriptors/fsl_sub.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "fsl_sub",
-  "tool-version": "1.1",
+  "tool-version": "2.0",
   "description": "A wrapper for FSL sub. Assumes that task will run on a file system shared with the submission process, that input files are already in place, and that output files will be handled by another process.",
-  "command-line": "fsl_sub [FSL_SUB_OPTIONS]",
+  "command-line": "/bin/bash .new-task-[SUBTASK_ID].sh",
```

Comment:
Works in our case but this completely bypass fsl_sub for tasks executed with METHOD=NONE.